cli/cli.go

```diff
@@ -174,11 +174,11 @@ func PrintCert(key, csrBytes, cert []byte) {
 }
 
 // PrintOCSPResponse outputs an OCSP response to stdout
+// ocspResponse is base64 encoded
 func PrintOCSPResponse(resp []byte) {
-	pemResponse := pem.EncodeToMemory(&pem.Block{Type: "CERTIFICATE REQUEST", Bytes: resp})
+	b64Resp := base64.StdEncoding.EncodeToString(resp)
```

Comment:
Thanks for the ping! I believe this won't affect the Boulder code base, since we call cfssl directly as a library rather than through the API, for both cert signing and OCSP signing. In case you're curious, here's the relevant section of code where we instantiate both signers: https://github.com/letsencrypt/boulder/blob/master/ca/certificate-authority.go#L126.